test/login_test.exs

```diff
@@ -162,6 +162,31 @@ defmodule LoginTest do
     end
   end
 
+  test "unix domain socket connection" do
```

Comment:
Should we document in readme that you need this to your hba file or should we assume it is there by default?

```
local all all trust
```